incubator/java-spring-boot2/image/project/update.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -x
+
+XTRACMDS=$*
+
```

Comment:
Could still test for 'dev' vs. 'final' or something, but I think stack-required packages, like xmlstartlet, should be in here, not in stack.yaml. Too central to how mechanics of the stack work.